app/src/main/java/com/teampophory/pophory/feature/home/store/StoreFragment.kt

```diff
@@ -34,56 +39,76 @@ class StoreFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        setupViewPager()
-        observeAlbumList()
+        initObserver()
         setSpannableString()
     }
 
-    private fun setupViewPager() {
-        //2차 스프린트를 위해 position 값을 받아둠
-        adapter = StoreAdapter(ItemDiffCallback(
-            onItemsTheSame = { old, new -> old == new },
-            onContentsTheSame = { old, new -> old == new }
-        )) { position ->
-            val intent = Intent(context, AlbumListActivity::class.java)
-            intent.putExtra("itemId", position)
-            requireContext().startActivity(intent)
+    private fun initObserver() {
+        viewModel.albums.observe(viewLifecycleOwner) { storeState ->
+            when (storeState) {
+                is StoreState.Uninitialized -> {
+                    viewModel.getAlbums()
+                    setupViewPager()
+                }
+
+                is StoreState.Loading -> {}
+
+                is StoreState.SuccessAlbums -> {
+                    with(binding) {
+                        storeAdapter?.submitList(storeState.data)
+
+                        //최초 데이터 세팅
+                        binding.tvStoreAlbumPhotoCount.text = storeState.data[0].photoCount.toString() + "/30"
```

Comment:
```suggestion
                        binding.tvStoreAlbumPhotoCount.text = storeState.data.firstOrNull().photoCount.toString() + "/30"
```

+) 앨범 없을때의 로직도 추가해야할듯

Reply:
아래 onPageChanged 콜백하고 동일 로직인 것 같은데 확인해보실?

Reply:
확인해보니 맞습니다 수정반영 했습니다!